.jshintrc

```diff
@@ -26,6 +26,7 @@
     "node": true,
 
     "globals": {
-        "fluid_2_0": true
+        "fluid_2_0": true,
```

Comment:
There shouldn't be any uses of the fluid_2_0 global in this codebase, and coding this here will only make us have to keep upgrading